### PR TITLE
gitignore python venv and benchmark results

### PR DESCRIPTION
**Pull request by laundmo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-.vscode
\ No newline at end of file
+.vscode
+
+# generated analysis html
+/analysis
+# python environment
+/venv
+# benchmark output json
+*.json
```
Comment (Voultapher) [inline .gitignore:10]: This would match all json files including sub-dirs. I'd prefer for it to be top-level only.

Comment (laundmo) [inline .gitignore:10]:
```suggestion
/*.json
```

Comment (laundmo) [reply .gitignore]: fixed by adding a leading slash

**Comment by Voultapher** [comment]
LGTM, thanks.

**Pull request merged**